notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -25,24 +25,32 @@ define([
          * @class KeyboardManager
          * @constructor
          * @param options {dict} Dictionary of keyword arguments :
-         *    @param options.events {$(Events)} instance 
+         *    @param options.events {$(Events)} instance
          *    @param options.pager: {Pager}  pager instance
          */
         this.mode = 'command';
+        this.view = 'tree';
         this.enabled = true;
         this.pager = options.pager;
         this.quick_help = undefined;
         this.notebook = undefined;
+        this.notebooklist = undefined;
         this.last_mode = undefined;
         this.bind_events();
         this.env = {pager:this.pager};
         this.actions = options.actions;
-        this.command_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env );
-        this.command_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
-        this.command_shortcuts.add_shortcuts(this.get_default_command_shortcuts());
-        this.edit_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
-        this.edit_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
-        this.edit_shortcuts.add_shortcuts(this.get_default_edit_shortcuts());
+        if (this.view === 'tree'){
+          this.tree_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
+          this.tree_shortcuts.add_shortcuts(this.get_default_tree_shortcuts());
+        }
+        if (this.view === 'notebook'){
```

Comment:
Actually since you set `this.view = 'tree'` on L32, this branch will never execute. Can you test to make sure that notebook view shortcuts have not regressed?